CMakeLists.txt

```diff
@@ -246,6 +246,7 @@ else()
 	include_directories("thirdparty/uuid/include/posix")
 endif()
 add_subdirectory("thirdparty/uuid")
+include_directories(thirdparty/ut)
```

Comment:
Is ut used within the main project(s)?

Reply:
No, but due to some cmake magic including in cmakelist.txt of nanofi itself wasn't enough. Not sure about the reasons, this way it worked.